raspberry-mint-dev/src/www/index.html

```diff
@@ -0,0 +1,630 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <title>Raspberry Mint development environment</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" href="./images/betty-192x192.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wdth,wght@0,62.5..100,100..900;1,62.5..100,100..900&display=swap"
+          rel="stylesheet">
+</head>
+<body>
+<div class="visually-hidden-focusable h4 text-center my-3">
+    <a href="#main-content">Skip to main content</a>
+</div>
+<div id="header" class="pt-1">
+    <div class="container">
+        <nav class="d-flex">
+            <div class="flex-grow-1">
+                <a class="header-site-logo h4" href="#" title="Go to the front page">
+                    <img src="./images/betty-192x192.png" alt="The Betty logo">
+                    Betty
+                </a>
+            </div>
+            <button id="toggle-nav-translations" class="btn btn-primary toggle-nav" type="button"
```

Comment:
Use responsive button sizes. They're too big on small screens now

Reply:
Make less attention grabbing. Do not use a primary color, and no default background.